lib/tasks/test_security.rake

```diff
@@ -20,18 +20,10 @@ namespace :test do
 
     desc "Run bundler audit"
     task :bundler_audit do
-      begin
-        require 'awesome_spawn'
-        puts AwesomeSpawn.run!("bundle-audit check", :params => {:update => nil, :verbose => nil}).output
-      rescue AwesomeSpawn::CommandResultError => err
-        puts "[#{err.result.command_line.inspect}] exited: [#{err.result.exit_status}] with:"
-        puts err.result.output
-        puts err.result.error
-        exit err.result.exit_status
-      end
+      exit $?.exitstatus unless system("bundle-audit check --update --verbose")
     end
   end
 
   desc "Run security tests"
-  task :security => %w(security:bundler_audit security:brakeman)
+  task :security => %w[security:bundler_audit security:brakeman]
```

Comment:
You shall not remove all of my `%w(...)`

haha, I don't really care but I don't prefer one so I typically just use parens.